components/AboutCard.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+
+interface Props {
+  bodyText: string;
+  imgURL: string;
+  reverse: boolean;
+  titleText: string;
+}
+
+export const AboutCard = ({ bodyText, imgURL, reverse, titleText }: Props) => {
+  if (!reverse) {
+    return (
+      <div className="flex w-full flex-col border">
+        <img
+          alt=""
+          className="aspect-video w-full object-cover"
+          src={imgURL}
+        ></img>
+        <div className="flex w-full flex-col justify-around gap-5 p-5 md:my-10 md:flex-row md:items-center">
+          <h2 className="md:w-1/5">{titleText}</h2>
+          <p className="md:w-3/5">{bodyText}</p>
+        </div>
+      </div>
+    );
+  } else {
+    return (
+      <div className="flex w-full flex-col border">
+        <div className="flex w-full flex-col justify-around gap-5 p-5 md:my-10 md:flex-row md:items-center">
+          <h2 className="md:w-1/5">{titleText}</h2>
+          <p className="md:w-3/5">{bodyText}</p>
+        </div>
+        <img
+          alt=""
+          className="aspect-video w-full object-cover"
+          src={imgURL}
+        ></img>
+      </div>
+    );
+  }
+};
+
+AboutCard.defaultProps = {
```

Comment:
I think we can remove the `defaultProps` on these components. Since the props are required, there's no need. If you do need default values, use the ES6 syntax instead: https://github.com/jsx-eslint/eslint-plugin-react/issues/2396#issuecomment-539184761